src/account.ts

```diff
@@ -51,3 +51,34 @@ export async function getAccountPhrase() {
     return;
   }
 }
+
+export async function signMessage(message: string): Promise<string> {
+  const wallet = await getWallet();
+
+  if (!wallet) {
+    throw 'No account';
+  }
+
+  return wallet.signMessage(message);
+}
+
+export async function signTransaction(
+  tx: providers.TransactionRequest
```

Comment:
It would be nice to provide a wrapper/abstraction around ether providers if possible. I don't love that this is coupling our external interface, to a current implementation detail (ethers). Even if all we do is re-export their type for now using our own name, that would give us a level of decoupling that could be really beneficial in the future if we decide to move more of the logic down to native code.

Reply:
that's completely cool, just used providers.TransactionRequest because it already existed, but if want to just re-export our own type that's fine. Their TransactionRequest type is a pretty standard ethereum transaction type, so happy to just re-export it